state-reconstruct-fetcher/src/types/v1.rs

```diff
@@ -101,15 +101,15 @@ impl TryFrom<&abi::Token> for V1 {
         for initial_calldata in initial_changes_calldata[4..].chunks(64) {
             let mut t = initial_calldata.array_chunks::<32>();
             let key = *t.next().ok_or_else(|| {
-                ParseError::InvalidCommitBlockInfo("initialStorageChanges".to_string())
+                ParseError::InvalidCommitBlockInfo("initialStorageChanges key".to_string())
```

Comment:
```suggestion
                ParseError::InvalidCommitBlockInfo("initialStorageChangesKey".to_string())
```
We should adhere to the pre-defined format. Ditto for the rest of the `ParseError`s

Reply:
Well, why should the code be a single word - is that some kind of Rust convention?

Reply:
Eh, not really. This is just what we have been doing up until this point

Reply:
Well OK, just to be consistent...